package org.ejercicios.basico.ManejoErrores.Uno;

public class Proceso {

    /*
        // Bloque 1
        try {
        // Bloque 2
        } catch (Exception error) {
        // Bloque 3
        }
        // Bloque 4
    * */

    /*
        // Bloque 1
        try {
        // Bloque 2
        } catch (ArithmeticException ae) {
        // Bloque 3
        } catch (NullPointerException ne) {
        // Bloque 4
        }
        // Bloque 5
    * */

    /*
        // Bloque1
        try {
        // Fragmento de código que puede
        // lanzar una excepción de tipo IOException
        // (p.ej. Acceso a un fichero)
        } catch (IOException error) {
        // Tratamiento de la excepción
        } finally {
        // Liberar recursos (siempre se hace)
        }
    * */
}
